sequel-pro.xcodeproj/project.pbxproj

```diff
@@ -2487,6 +2505,7 @@
 			isa = PBXGroup;
 			children = (
 				584D876815140D3500F24774 /* SPMySQL.framework */,
+				8B6FACFE1C86035800F2805A /* SPMySQL Unit Tests.xctest */,
```

Comment:
I think this has been included in another commit in the meantime and can be removed (same for next change)